lusid-sdk-python/tests/business_agility_test_environment.py

```diff
@@ -0,0 +1,662 @@
+import unittest
+import uuid
+import lusid.models as models
+import pytz
+from datetime import datetime, timedelta
+from finbournetest import TestFinbourneApi, timeit
+
+try:
+    # Python 3.x
+    from urllib.request import pathname2url
+except ImportError:
+    # Python 2.7
+    from urllib import pathname2url
+
+
+class businessAgilityTestEnvironment(TestFinbourneApi):
+
+    @timeit
+    def import_data(self):
+        """
+        We are an asset manager who has recently secured a contract to work with a new small pension fund. The pension
```

Comment:
Move the introductory test to the top of the test case

Reply:
Complete